webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx

```diff
@@ -3,27 +3,21 @@
 
 import classNames from 'classnames';
 import React from 'react';
-import {defineMessage} from 'react-intl';
-
-import LocalizedIcon from 'components/localized_icon';
+import {useIntl} from 'react-intl';
 
 type Props = {
     additionalClassName?: string;
 }
 
-const iconTitle = defineMessage({
-    id: 'generic_icons.previous',
-    defaultMessage: 'Previous Icon',
-});
-
 const PreviousIcon = ({additionalClassName}: Props) => {
+    const {formatMessage} = useIntl();
+    
```

Comment:
It is not shown here but you are adding 4 spaces and the linter is not liking it. Just keep the empty line completely empty.

Reply:
@jespino, Sure, I will remove the spaces on line 14.